client/src/components/StarReview.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const StarReview = ({ rating }) => {
+  const stars = [];
+
+  const totalStars = 5;
+  // Add a full star for every whole number
+  for (let i = 0; i < Math.floor(rating); i++) {
+    stars.push(<div className="star-review__star star-review__star--full" key={i}><img src="#" alt="Full star" /></div>);
+  }
+  // If decimal between .5 and 1, show half-star graphic
+  if ((rating - stars.length) >= .5) {
+    stars.push(<div className="star-review__star star-review__star--half" key={1}><img src="#" alt="Half star" /></div>);
+  }
```

Comment:
Business requirements docs specify stars have to come in 1/4 and 3/4 full, also.
(See top of page 4, https://docs.google.com/document/d/1veFXCieet4051fKT2Lsgx0AVnjHdVZXTE-cnTn6rb50/edit)